### PR TITLE
MOTOR-476: Update FLE documentation for community version demonstrating explicit encryption/decryption

### PR DESCRIPTION
**Pull request by guanlinzhou**
Ported over examples for explicit encryption/decryption from PyMongo CSFLE. Built docs to check that they render correctly and ran each example to ensure that they work as expected.

### CODE REVIEW

File: doc/examples/encryption.rst [hunk 1]
```diff
@@ -91,10 +91,34 @@ application from inserting unencrypted values for any fields marked with the
 
 The following example shows how to setup automatic client-side field level
 encryption using
-:class:`~pymongo.encryption.ClientEncryption` to create a new encryption
+:class:`~motor.motor_asyncio.AsyncIOMotorClientEncryption` to create a new encryption
 data key and create a collection with the
 `Automatic Encryption JSON Schema Syntax
 <https://dochub.mongodb.org/core/client-side-field-level-encryption-automatic-encryption-rules>`_.
 
 .. literalinclude:: server_fle_enforcement_example.py
```
Comment (ShaneHarvey) [inline doc/examples/encryption.rst:99]: Can you also update the previous two examples to use AsyncIOMotorClientEncryption instead of ClientEncryption?

Comment (guanlinzhou) [reply doc/examples/encryption.rst:99]: yep, got it!

**Review by ShaneHarvey — APPROVED** [review]
Nice! LGTM

**Review by prashantmital — APPROVED** [review]
Nicely done! LGTM.

**Pull request merged**